tribe-ext-extension-template.php

```diff
@@ -204,6 +222,48 @@ private function php_version_check() {
 			return true;
 		}
 
+		/**
+		 * Check if we have the needed view. Admin notice if we don't and user should see it.
+		 *
+		 * @return bool
+		 *
+		 * @todo Remove method if extension works with both V1 and V2 views
+		 */
+		private function v2_views_check() {
+			$view_required_version = $this->$required_view;
+
+			$view_breaks_version = $view_required_version == 'V1 legacy' ? 'V2 updated' : 'V1 legacy';
+
+			$show_warning = (
+				$view_required_version === 'V1 legacy' && tribe_events_views_v2_is_enabled()
+				|| $view_required_version === 'V2 updated' && ! tribe_events_views_v2_is_enabled()
+			)
+				? true
+				: false;
```

Comment:
avoid ternaries, especially multi-line, to be more legible -- and each condition on its own line